policyengine_us/variables/gov/states/nj/tax/exemptions/nj_regular_exemption.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class nj_regular_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NJ regular exemption"
```

Comment:
I know we're inconsistent on this, but I'd like to move to spelling states out
```suggestion
    label = "New Jersey regular exemption"
```

Reply:
Got it